multiaccounts/settings/events.go

```diff
@@ -0,0 +1,6 @@
+package settings
+
+type EventSettingChanged struct {
```

Comment:
I'm still not following how it this one different from `SettingChangedEvent` in `services/accounts/settings.go`? 🤔 
Are both really in use?